misc/python/materialize/cloudtest/app/application.py

```diff
@@ -36,11 +36,14 @@ def coverage_mode(self) -> bool:
     def acquire_images(self) -> None:
         raise NotImplementedError
 
-    def kubectl(self, *args: str) -> str:
+    def kubectl(self, *args: str, namespace: Optional[str] = None) -> str:
```

Comment:
Not sure if it is possible to specify `namespace: str = self.namespace()` since the default is constant. Without having tested it, I am sceptical if Python will accept it. Anyway, it will most likely not contian the correct value if `self.namespace()` is overridden in a subclass because the default is initialized only once.

However, I am happy to add an else case to `is not None` in the method with this value.

Reply:
Please note that this is in `Application`. This class does currently not store any namespace.